src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -215,6 +247,9 @@ export const generateContent: TranslationFn = content => {
   const displayState = currentStateFn(userCase).at(
     (userCase.state === State.OfflineDocumentReceived ? userCase.previousState : userCase.state) as State
   );
+  const hasApplicantAppliedForFinalOrder = isApplicant2
+    ? userCase.applicant2AppliedForFinalOrderFirst === YesOrNo.YES
+    : userCase.applicant1AppliedForFinalOrderFirst === YesOrNo.YES;
```

Comment:
think a better name for the variable would be hasApplicantAppliedForFinalOrderFirst

Reply:
yh, tbh im only using that field as it seems its the only way to determine if they have applied for final order in joint.